app/src/main/java/com/nitrr/ecell/esummit/ecellapp/misc/Animation/LoginAnimation.java

```diff
@@ -0,0 +1,160 @@
+package com.nitrr.ecell.esummit.ecellapp.misc.Animation;
+
+import android.animation.ObjectAnimator;
+import android.content.Context;
+import android.transition.Fade;
+import android.view.View;
+import android.widget.Button;
+import android.widget.ImageButton;
+import android.widget.ImageView;
+import android.widget.LinearLayout;
+import android.widget.TextView;
+
+import com.nitrr.ecell.esummit.ecellapp.R;
+import com.nitrr.ecell.esummit.ecellapp.activities.Login;
+
+public class LoginAnimation {
+
+    LinearLayout upperlinearLayout;
+    LinearLayout lowerLinearLayout;
+    ImageView lowerPoly;
+    ImageView upperPoly;
+    ImageView  registerScreenTransitionArrow;
+    ImageView signinScreenTransitionArrow;
+    ImageButton google;
+    ImageButton facebook;
+    ImageButton registerTransitionButtoBackground;
+    ImageButton signinTransitionButtonBackground;
+    TextView forgot;
+    TextView registertext;
+    TextView signinText;
+    Button signin;
+    Button register;
+
```

Comment:
Buttons can be initalized as Button signIn, register;
Do it for every common variable.

Reply:
Done!